src/index.js

```diff
@@ -37,10 +37,14 @@ module.exports = options => ({
       )
       return extractedCSS
     } catch (e) {
+      const name = e.name
       // incorrect interpolations will throw CssSyntaxError and they'll be handled by stylelint
       if (e.name === 'CssSyntaxError') {
         throw e
       }
+      if (name === 'SyntaxError') {
```

Comment:
Adding a comment above here would be awesome too

Reply:
Will add a comment and update PR

Reply:
@ericberens By the way, how about checking `e.name` directly, without a new local variable declared?

Reply:
I agree, especially because we use `e.name` in the conditional above, that's odd and inconsistent

Reply:
I'm wondering if it's necessary to even do the `name === 'SyntaxError'` check, will it be sufficient to just `return input` on all errors and let the stylelint engine process it?